From 875ed362e8ec10d1880af741d7e082dcebd82200 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matt=C3=A9o=20Rossillol=E2=80=91=E2=80=91Laruelle?=
 <beatussum@protonmail.com>
Date: Fri, 22 Jan 2021 13:28:50 +0100
Subject: [PATCH] fix `gtk-update-icon-cache` call
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Call `gtk-update-icon-cache` during `pkg_postrm()` instead of during
`src_compile()`.

Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
---
 install.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/install.sh b/install.sh
index be3c9493..ce461fb4 100755
--- a/install.sh
+++ b/install.sh
@@ -115,8 +115,6 @@ install_theme() {
   ln -sr "${THEME_DIR}/24"                                                       "${THEME_DIR}/24@3x"
   ln -sr "${THEME_DIR}/32"                                                       "${THEME_DIR}/32@3x"
   ln -sr "${THEME_DIR}/scalable"                                                 "${THEME_DIR}/scalable@3x"
-
-  gtk-update-icon-cache "${THEME_DIR}"
 }
 
 while [ $# -gt 0 ]; do
-- 
2.26.2

